.github/workflows/ci-build.yml

```diff
@@ -125,12 +127,14 @@ jobs:
       with:
         fetch-depth: 0
 
+    - run: printenv
+
     - uses: actions/checkout@v3
       name: Checkout private submoules recursively
       id: private
       if: ${{ inputs.SUBMODULES_RECURSIVE  == true && inputs.SUBMODULES_PRIVATE == true }}
       with:
-        ssh-key: ${{ secrets.GH_BOT_DEPLOY_KEY }}
+        token: ${{ secrets.CI_PAT }}
```

Comment:
just for testing, I'm not actually proposing we eliminate SSH, but wanted a quick test of http auth with a PAT